support-frontend/assets/helpers/campaigns.jsx

```diff
@@ -18,92 +17,48 @@ export type CampaignSettings = {
   backgroundImage?: string,
   goalReachedCopy: React$Element<string> | null,
   localCurrencySymbol: string,
+  extraComponent?: React$Element<string>,
```

Comment:
Is this the right type for the extra component?

Reply:
Yes, it is the right type though I don't fully understand the `<string>` part -- @tomrf1 is there a simple way of saying how `<string>` works in the context of a type?

Reply:
Good question, I'm pretty sure this type parameter is the props type. So actually, should it just be `React$Element<>`?